docs/user/icl/ssh-intro.rst

```diff
@@ -0,0 +1,67 @@
+.. _basic SSH introduction:
+
+SSH
+^^^
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Artemis uses SSH as a simple way for users to connect securely to repositories to perform Git operations.
+
+What is SSH?
+""""""""""""
+
+.. _SSH (Secure Shell): https://en.wikipedia.org/wiki/Secure_Shell
+
+`SSH (Secure Shell)`_ is a protocol that allows you to securely connect to another computer over a network.
+It’s mostly used by system administrators, developers, and IT professionals to remotely manage servers or computers.
+SSH provides a secure, encrypted communication channel between your computer and a remote machine, so any data passed (like passwords or commands) is protected from eavesdropping.
+In Artemis you cna use SSH to access your repositories with Git.
+
+Why use SSH?
+""""""""""""
+
+The main advantage of SSH is security.
+When you connect to a remote machine using SSH, all the data exchanged between your computer and the server is encrypted.
+This means if someone tries to intercept the communication, they can't read it. It’s like sending messages through a locked box that only you and the server can open.
+
+How does SSH work?
+""""""""""""""""""
+
+SSH works by using two components:
+
+- Client: The computer you are using to connect.
+- Server: The machine you want to connect to.
+
+When you want to connect, your SSH client sends a request to the server.
+If the connection is successful, you can log in to the server and start working as if you were sitting in front of it.
+The connection uses SSH keys for authentication.
```

Comment:
I think this is not technically correct, you can also use username and password for starting a SSH connection afaik